types/react-native-share-menu/index.d.ts

```diff
@@ -1,12 +1,35 @@
-// Type definitions for react-native-share-menu 2.2
+// Type definitions for react-native-share-menu 5.0
 // Project: https://github.com/meedan/react-native-share-menu#readme
-// Definitions by: Haseeb Majid <https://github.com/hmajid2301>
```

Comment:
You should add your name underneath no need to remove mine.

Reply:
Done. Thanks for the guidance.